ngx_http_fancyindex_module.c

```diff
@@ -1360,7 +1372,11 @@ ngx_http_fancyindex_cmp_entries_name_desc(const void *one, const void *two)
     ngx_http_fancyindex_entry_t *first = (ngx_http_fancyindex_entry_t *) one;
     ngx_http_fancyindex_entry_t *second = (ngx_http_fancyindex_entry_t *) two;
 
-    return (int) ngx_strcmp(second->name.data, first->name.data);
+    if(case_sensitive) {
+        return (int) ngx_strcmp(second->name.data, first->name.data);
+    } else {
+        return (int) ngx_strcasecmp(second->name.data, first->name.data);
+    }
```

Comment:
The rest of this file always uses a space after an `if` statement so I suggest doing that here too:

```suggestion
    if (case_sensitive) {
        return (int) ngx_strcmp(second->name.data, first->name.data);
    } else {
        return (int) ngx_strcasecmp(second->name.data, first->name.data);
    }
```

Some `if` and `else` statements in this file that are only followed by one statement omit the curly braces; if that is the desired style of this file, I suggest:

```suggestion
    if (case_sensitive)
        return (int) ngx_strcmp(second->name.data, first->name.data);
    else
        return (int) ngx_strcasecmp(second->name.data, first->name.data);
```

Personally I find it clearer when curly braces are always used but the owner of this repository can say which style they prefer.

Reply:
I also have a tendency to prefer always using curly braces, to avoid ambiguity.